app/views/recipes/edit.html.haml

```diff
@@ -6,6 +6,8 @@
   %br
   = form.text_field :title
   %br
+  = form.collection_select :category_id, @categories, :id, :name
```

Comment:
Not seen `collection_select` before. Can you point me to some documentation?

Reply:
http://guides.rubyonrails.org/form_helpers.html 
from 3.3